pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools>=64", "setuptools-git-versioning", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "bokeh_fastapi"
+dynamic = ["version"]
+description = "A web application using Bokeh and FastAPI"
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+    { name = "Philipp Rudiger", email = "prudiger@anaconda.com" }
+]
+license = {file = "LICENSE.txt"}
+dependencies = [
+    "fastapi>=0.68.0",
+    "bokeh>=2.3.3",
+    "uvicorn>=0.15.0"
```

Comment:
Do we really want to hard depend on that? FastAPI does not either. They just recommend `uvicorn`. I would do it the same and maybe add a `pip install bokeh-fastapi[serve]` extra to install everything we need to serve.

Reply:
I wasn't aware, will defer to you on this if it makes sense to have `FastAPI` without `uvicorn`. Can we use another server as well?

Reply:
> Can we use another server as well?

Yes: https://asgi.readthedocs.io/en/latest/implementations.html#servers. uvicorn is by far the most popular, but since we want users to also use their own FastAPI instance, we should also allow them to bring their own server.